server/services/node-red/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "gladys-node-red",
+  "version": "1.0.0",
+  "main": "index.js",
+  "os": [
+    "darwin",
+    "linux",
+    "win32"
+  ],
+  "cpu": [
+    "x64",
+    "arm",
+    "arm64"
+  ],
+  "dependencies": {
```

Comment:
Check all deps, seems to be a duplicate

Re-run npm install after to update package-lock.json